packages/components/src/components/SquiggleViewer/ExpressionViewer.tsx

```diff
@@ -202,10 +202,15 @@ export const ExpressionViewer: React.FC<Props> = ({ value }) => {
           value={value}
           heading="Plot"
           renderSettingsMenu={({ onChange }) => {
-            let disableLogX = plot.distributions.some((x) => {
-              let pointSet = x.distribution.pointSet(environment);
-              return pointSet.ok && hasMassBelowZero(pointSet.value.asShape());
-            });
+            const disableLogX =
+              plot.tag === "distributions"
+                ? plot.distributions.some((x) => {
+                    let pointSet = x.distribution.pointSet(environment);
+                    return (
+                      pointSet.ok && hasMassBelowZero(pointSet.value.asShape())
+                    );
+                  })
+                : true; // TODO - check min/max values for fn plots and support log scale in FunctionChart
```

Comment:
just to be clear, this is a TODO for later?

Reply:
Log scales were never implemented for function charts, so it's fine to do it later, but also not hard to do in this PR, I'll probably do it.

Reply:
Wait, this setting is in distribution section, so for function charts it works for distributions that show on mouseover. So it should be true, without `TODO`.